PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
+
+    #URL = "https://api.develop.socotra.com/account/authenticate"
+    URL = "http://127.0.0.1:8080/account/authenticate"
```

Comment:
By convention, variable names in all caps are assumed to be defined in the global scope. This isn't.